jme3-terrain/src/main/java/com/jme3/terrain/geomipmap/TerrainPatch.java

```diff
@@ -202,9 +202,9 @@ public void generateLodEntropies() {
             IndexBuffer idxB = geomap.writeIndexArrayLodDiff(curLod, false, false, false, false, totalSize);
             Buffer ib;
             if (idxB.getBuffer() instanceof IntBuffer)
-                ib = (IntBuffer)idxB.getBuffer();
+                ib = idxB.getBuffer();
             else
-                ib = (ShortBuffer)idxB.getBuffer();
+                ib = idxB.getBuffer();

```

Comment:
This code looks useless. Why couldn't `ib` be a `ByteBuffer` or a `LongBuffer`? Specifically when we already use our wrapper `IndexBuffer`

Reply:
Good point. I'll remove the test.